post_installation_configuration/post-install-storage-configuration.adoc

```diff
@@ -26,61 +26,23 @@ cluster to your requirements, including storage configuration.
-
-include::modules/dynamic-provisioning-azure-disk-definition.adoc[leveloffset=+2]
-
-include::modules/dynamic-provisioning-azure-file-definition.adoc[leveloffset=+2]
-
-include::modules/dynamic-provisioning-azure-file-considerations.adoc[leveloffset=+3]
-
-include::modules/dynamic-provisioning-gce-definition.adoc[leveloffset=+2]
+Dynamic Provisioning allows you to create storage volumes on-demand, eliminating the need for cluster administrators to pre-provision storage. See xref:../storage/dynamic-provisioning.adoc#dynamic-provisioning[Dynamic provisioning].
 
 // include::modules/dynamic-provisioning-gluster-definition.adoc[leveloffset=+2]
 
 // include::modules/dynamic-provisioning-ceph-rbd-definition.adoc[leveloffset=+2]
 
-include::modules/dynamic-provisioning-vsphere-definition.adoc[leveloffset=+2]
-
-include::modules/dynamic-provisioning-change-default-class.adoc[leveloffset=+1]
-
-[id="post-install-optimizing-storage"]
-== Optimizing storage
-
-Optimizing storage helps to minimize storage use across all resources. By
-optimizing storage, administrators help ensure that existing storage resources
-are working in an efficient manner.
-
-include::modules/available-persistent-storage-options.adoc[leveloffset=+1]
-
 include::modules/recommended-configurable-storage-technology.adoc[leveloffset=+1]
 
-[role="_additional-resources"]
-.Additional resources
-
-* xref:../scalability_and_performance/recommended-performance-scale-practices/recommended-etcd-practices.adoc#recommended-etcd-practices[Recommended etcd practices]
-
-
 [id="post-install-deploy-OCS"]
 == Deploy Red Hat OpenShift Data Foundation
 // This section is sourced from storage/persistent_storage/persistent-storage-ocs.adoc
-{rh-storage-first} is a provider of agnostic persistent storage for {product-title} supporting file, block, and object storage, either in-house or in hybrid clouds. As a Red Hat storage solution, {rh-storage-first} is completely integrated with {product-title} for deployment, management, and monitoring.
 
-include::modules/deploy-red-hat-openshift-container-storage.adoc[leveloffset=+1]
+{rh-storage-first} is a provider of agnostic persistent storage for {product-title} supporting file, block, and object storage, either in-house or in hybrid clouds. As a Red Hat storage solution, {rh-storage-first} is completely integrated with {product-title} for deployment, management, and monitoring. For more information, see the link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation[{rh-storage-first} documentation].
+
+[IMPORTANT]
+====
+{rh-storage} on top of Red Hat Hyperconverged Infrastructure (RHHI) for Virtualization, which uses hyperconverged nodes that host virtual machines installed with {product-title}, is not a supported configuration. For more information about supported platforms, see the link:https://access.redhat.com/articles/4731161[Red Hat OpenShift Data Foundation Supportability and Interoperability Guide].
```

Comment:
I'm not thrilled with this note, as the parenthetical phrase makes it wordy and might not be needed. But, I see that it is existing text, previously approved. 

```suggestion
{rh-storage} on top of Red Hat Hyperconverged Infrastructure (RHHI) for Virtualization is not a supported configuration. For more information about supported platforms, see the link:https://access.redhat.com/articles/4731161[Red Hat OpenShift Data Foundation Supportability and Interoperability Guide].
```

Reply:
Yep, gonna leave it as it is for now, since it's pre-existing and also used in 2 sections